kolibri/plugins/coach/assets/src/app.js

```diff
@@ -57,6 +57,13 @@ class CoachToolsModule extends KolibriModule {
           },
         },
         {
+          name: PageNames.COACH_EXERCISE_RENDER_PAGE,
+          path: '/exercise-render',
+          handler: (toRoute, fromRoute) => {
+            actions.showCoachExerciseRenderPage(store);
+          },
+        },
+        {
```

Comment:
We don't want to actually add a router URL, action, and page name for this component. It should be stand-alone and re-usable.

If you need a place for testing, we'd previously used the `scratchpad` for this.